sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/Generated/Models/MethodRequest.cs

```diff
@@ -17,7 +17,6 @@ public abstract partial class MethodRequest
         /// <summary> Initializes a new instance of MethodRequest. </summary>
         protected MethodRequest()
         {
-            ApiVersion = "1.1";
```

Comment:
why was this removed?

Reply:
Same reason as above

Reply:
Well this is in a protected ctor with a discriminator, it never works anyway (because we will not call this ctor, we only call the internal ctor on the derived types).
But I will introduce this back using some customized code.